### DISCUSSION TITLE
Waiting since the July

### DISCUSSION THREAD

**Discussion by VitaliyRDev** [community/community#7382] [Copilot Conversations]
Hello, i'm waiting on Github Copilot to test it since the July, but still didn't get any answer. What's the problem ?

**Comment by IllusionTheDev** [comment]
I believe that Copilot's waitlist takes into account your activity, if you do a lot of commits, handle a lot of issues / pull requests, you get access earlier.

I personally got my access within 24 hours, those are my tips for you.

**Reply by uBaby4life** [reply]
I have applied at June and it has been three days since I got it and I have 0 GitHub activity. I did something like this - I joined the waitlist at June and then to check the status, I have reapplied again my signing up (they didn't accept my sign up since I was already signed up). At the same time I used another email that I have 0 GitHub activity and signed up at the same day (when I was checking the update). And I got a mail Welcoming me to the Copilot within 72 hours after that both on my account.

**Comment by slpkbt** [comment]
I don't know how they check accounts and approve, but I really want to use copilot. I have a friend who got accepted in a day, another friend with an active GitHub has been waiting a month.

I'm on the waitlist right now, too, and I'm really waiting for approval.
